root/utility/isDisabledLink.js

```diff
@@ -8,12 +8,19 @@
  */
 
 import isGreyedOut from '../url/utility/isGreyedOut';
+import isFutureDate from '../utility/isFutureDate';
 
 export default function isDisabledLink(
-  relationshipOrLinkDatePeriod: {+ended: boolean, ...},
+  relationshipOrLinkDatePeriod: {
+    +end_date: PartialDateT | null,
+    +ended: boolean,
+    ...},
```

Comment:
(not sure if eslint complains about this, but just in case)
```suggestion
    ...
  },
```